### DISCUSSION TITLE
Timestep reduction

### DISCUSSION THREAD

**Discussion by DanRRRR** [epochpic/epoch#643] [General]
EPOCH chooses timestep by itself. Is it possible to push it to multiply timestep by user defined factor from its own decision ?

**Comment by TomGoffrey** [comment]
EPOCH calculates the time step according to the CFL condition. That value is then multiplied by `dt_multiplier` (in the Control block of the input deck). Does that do what you want?

**Comment by DanRRRR** [comment]
Thanks, Tom, will check, most probably this is exactly what i needed. The idea is to check how timestep influences the self-heating which at higher densities could be very large.
